tests/test_stocks_model.py

```diff
@@ -352,3 +352,40 @@ def test_bucket_value_on():
     with pytest.raises(Exception):
         config.value_on("2016-06-01")
     assert config.value_on("2016-06-08") == quantity * value
+
+
+@pytest.mark.django_db(transaction=True)
+def test_bucket_historical():
+    """
+    Tests InvestmentBucket.historical()
+    """
+    user = User.objects.create(username='user1', password="a")
+    stock = Stock(
+        name="Name1X",
+        ticker="TKRC"
+    )
```

Comment:
This will trigger a signal to fetch stock quotes. Disconnect it.

Reply:
Good call, thanks.

Reply:
wait isn't it disconnected by the setup/teardown already?